src/App.js

```diff
@@ -11,35 +17,152 @@ class App extends React.Component {
       cardDeck: makeShuffledDeck(),
       // currCards holds the cards from the current round
       currCards: [],
+      cardsLeft: 52,
+      gameStarted: false,
+
+      playerOneCurrWinner: false,
+      playerOneScore: 0,
+      playerTwoScore: 0,
     };
   }
 
+  playerOneIsHigher = (list) => {
+    const suits = ["Diamonds", "Clubs", "Hearts", "Spades"];
+    if (list.length === 0) {
+      console.log(list);
+      return null;
+    }
+    if (list[0].rank === list[1].rank) {
+      if (suits.indexOf(list[0].suit) > suits.indexOf(list[1].suit)) {
+        return true;
+      } else {
+        return false;
+      }
+    } else if (list[0].rank > list[1].rank) {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
   dealCards = () => {
     // this.state.cardDeck.pop() modifies this.state.cardDeck array
     const newCurrCards = [this.state.cardDeck.pop(), this.state.cardDeck.pop()];
+    const newPlayerOneCurrWinner = this.playerOneIsHigher(newCurrCards);
     this.setState({
       currCards: newCurrCards,
+      cardsLeft: this.state.cardsLeft - 2,
+      gameStarted: true,
+      playerOneCurrWinner: newPlayerOneCurrWinner,
     });
+    if (newPlayerOneCurrWinner) {
+      this.setState({
+        playerOneScore: this.state.playerOneScore + 1,
+      });
+    } else {
+      this.setState({
+        playerTwoScore: this.state.playerTwoScore + 1,
+      });
+    }
```

Comment:
I think you should consolidate these into a single state update. You could look up how to conditionally set a object property